nibabel/freesurfer/io.py

```diff
@@ -160,6 +160,29 @@ def read_morph_data(filepath):
     return curv
 
 
+def write_morph_data(filepath, values):
+    """Write out a Freesurfer morphometry data file.
+
+    See:
+    http://www.grahamwideman.com/gw/brain/fs/surfacefileformats.htm#CurvNew
+
+    Parameters
+    ----------
+    filepath : str
+        Path to annotation file to be written
+    values : ndarray, shape (n_vertices,)
+        Surface morphometry values
+    """
+    magic_bytes = np.array([255, 255, 255], dtype=np.uint8)
+    with open(filepath, 'wb') as fobj:
+        magic_bytes.tofile(fobj)
+
+        # vertex count, face count (unused), vals per vertex (only 1 supported)
+        np.array([len(values), 0, 1], dtype='>i4').tofile(fobj)
+
+        np.array(values, dtype='>f4').tofile(fobj)
```

Comment:
Just possible (if this has been loaded off disk for example) array is already in correct format, in which case you can avoid copy by `np.asarray(values).astype('>f4').tofile(fobj)`.